python/setup.py

```diff
@@ -235,6 +235,9 @@ def get_packages(self):
         "adag": [
             "cupy-cuda12x; sys_platform != 'darwin'",
         ],
+        "cg": [
+            "torch-cuda12x; sys_platform != 'darwin'",
```

Comment:
```
$ pip install torch-cuda12x
ERROR: Could not find a version that satisfies the requirement torch-cuda12x (from versions: none)
ERROR: No matching distribution found for torch-cuda12x
```

there is no such pypi package on public pypi.org

Reply:
https://pypi.org/project/torch-cuda12x/ this page does not exist.

Reply:
typo, fixed, thanks for catching